### ISSUE TITLE
Can't select text inside chat

### ISSUE THREAD

**Issue by DamianX** [space-wizards/space-station-14#510]
How am I going to copypaste the one guy saying "lol lmao" IC into adminhelps?
Needs to be implemented.

**Comment by Ephememory** [comment]
should we create a base class TextSelectable that text entires like the chatbox output can derive from? seems like https://github.com/space-wizards/RobustToolbox/blob/master/Robust.Client/UserInterface/Controls/LineEdit.cs#L83 just has the functionality of selecting text snowflaked in.

**Comment by PJB3005** [comment]
Solving this with extra layers of inheritance is *not* the way to go. That's all I have to say on that.

**Comment by Ephememory** [comment]
> Solving this with extra layers of inheritance is _not_ the way to go. That's all I have to say on that.

yeah lets make a TextSelectableSystem and TextSelectable component
